src/signal-processing/ns_speex_tilde

```diff
@@ -0,0 +1,167 @@
+/**
+@file ns_speex_tilde.c
+@author Dennis Guse, Frank Haase, Michal Soloducha
+@date 2016-10-21
+@license GPLv3 or later
+
+ns_speex~ applies the _noise suppression_ algorithm of [Speex](http://www.speex.org/).
+
+Parameters:
+  ns_speex~ FRAME_SIZE SAMPLE_RATE MAX_NOISE_ATT
+  FRAME_SIZE: 80, 160, 240, 320, 640 [samples]
+  SAMPLE_RATE: 8000, 16000, 32000 [Hz]
+  MAX_NOISE_ATT - maximum noise attenuation (negative number) [dB]
```

Comment:
@MAX_NOISE_ATT

I would prefer the long version (MAX_NOISE_ATTENUATION) and also the expected range -100..0?
What is the default value?